packages/neon/neon/lib/src/utils/global_popups.dart

```diff
@@ -7,37 +7,65 @@ class GlobalPopups {
   static GlobalPopups? _instance;
   bool _registered = false;
 
-  void register(final BuildContext context) {
+  final subscriptions = <StreamSubscription>[];
+
+  late BuildContext context;
+
+  Future<void> register(final BuildContext context) async {
+    this.context = context;
     if (_registered) {
       return;
     }
+    _registered = true;
 
-    final globalOptions = Provider.of<GlobalOptions>(context, listen: false);
     final firstLaunchBloc = Provider.of<FirstLaunchBloc>(context, listen: false);
     final nextPushBloc = Provider.of<NextPushBloc>(context, listen: false);
+    final accountsBloc = Provider.of<AccountsBloc>(context, listen: false);
+    final account = accountsBloc.activeAccount.value!;
+    final appsBloc = accountsBloc.activeAppsBloc;
```

Comment:
This won't work with multiple accounts because the second register call will not reach this point.

Reply:
Ah yes;
I thought about that but probably forgot it again :)

Would it make sense to have a second method that registers those or just a separate singleton?

Reply:
IMO the purpose of global is to be above any context like an account or app. This kind of defeats the purpose. Maybe a different place is better? You could make another similar singletone `AccountPopups` that gets registered in the home page as well.